EIPS/Increase block rewards to 5 ETH.md

```diff
@@ -0,0 +1,34 @@
+---
+eip: <to be assigned>
+title: Increase block rewards to 5 ETH
+author: Ben Tinner (@Terra854)
+discussions-to: https://www.reddit.com/r/ethereum/comments/k4252u/thoughts_about_increasing_block_rewards_till_we/
+status: Draft
+type: Standards Track
+category Core
+created: 2020-12-01
+---
+
+## Simple Summary
+This EIP will increase the block reward paid to proof-of-work (POW) miners, giving them more opporturnities to acquire the minimum number of ETH needed to run a standalone proof-of-stake (POS) validator node before full implementation of ETH 2.0
+
+## Abstract
+By increasing the block rewards back to the original 5 ETH when the network first started, it will convince POW miners to stay on mining Ethereum without worrying too much on profitability. It will also give smaller POW miners more opporturnities to be able to acquire the minimum amount of ETH needed to become a POS validator before we transition fully to ETH 2.0
```

Comment:
This should be moved to the Motivation section.  The `Abstract` should be a short, human readable, version of the specification.  The Motivation section is where you want to talk about *why* this EIP is a good idea.